de.fhdo.lemma.servicedsl.extractor/src/de/fhdo/lemma/servicedsl/extractor/ServiceDslExtractor.xtend

```diff
@@ -0,0 +1,272 @@
+package de.fhdo.lemma.servicedsl.extractor
+
+import de.fhdo.lemma.data.ComplexType
+import de.fhdo.lemma.data.PrimitiveType
+import de.fhdo.lemma.service.Endpoint
+import de.fhdo.lemma.service.Import
+import de.fhdo.lemma.service.ImportType
+import de.fhdo.lemma.service.ImportedType
+import de.fhdo.lemma.service.Interface
+import de.fhdo.lemma.service.Microservice
+import de.fhdo.lemma.service.MicroserviceType
+import de.fhdo.lemma.service.Operation
+import de.fhdo.lemma.service.Parameter
+import de.fhdo.lemma.service.ServiceModel
+import de.fhdo.lemma.service.Visibility
+import de.fhdo.lemma.technology.CommunicationType
+import de.fhdo.lemma.technology.ExchangePattern
+import de.fhdo.lemma.technology.Protocol
+import de.fhdo.lemma.service.ImportedServiceAspect
+
+
+/**
+ * Model-to-text extractor for the Service DSL.
+ *
+ * @author <a href="mailto:jonas.sorgalla@fh-dortmund.de">Jonas Sorgalla</a>
+ */
+class ServiceDslExtractor {
+    static val IMPORTED_TECH_ALIASES = <String>newArrayList()
```

Comment:
Remove the round brackets after `newArrayList` to be consistent with the rest of the file.